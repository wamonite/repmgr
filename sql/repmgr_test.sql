select * from repmgr_update_standby_location('');
select * from repmgr_get_last_standby_location();
select * from repmgr_update_last_updated();
select * from repmgr_get_last_updated();
