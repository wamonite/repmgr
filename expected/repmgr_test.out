select * from repmgr_update_standby_location('');
 repmgr_update_standby_location 
--------------------------------
 f
(1 row)

select * from repmgr_get_last_standby_location();
 repmgr_get_last_standby_location 
----------------------------------
 
(1 row)

select * from repmgr_update_last_updated();
 repmgr_update_last_updated 
----------------------------
 
(1 row)

select * from repmgr_get_last_updated();
 repmgr_get_last_updated 
-------------------------
 
(1 row)

